soatest/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM parasoft/soavirt
 
 ARG HOST_UID=995
 ARG HOST_GID=991
+ARG CHROME_VERSION=112.0.5615.49
```

Comment:
It doesn't look like this CHROME_VERSION argument is actually being used in the Dockerfile.  The Dockerfile should be cleaned up to use this argument.